Umbraco-Cloud/Databases/Backups/index.md

```diff
@@ -0,0 +1,67 @@
+---
+versionFrom: 7.0.0
+---
+
+# Database backups
+Sometimes you might need to have a backup of your Cloud database. This can be accomplished in at lest two ways. This article will go through two of those options.
```

Comment:
```suggestion
Sometimes you might need to have a backup of your Cloud database. This can be accomplished in at least two ways. This article will go through two of those options.
```